tap_amazon_ads/exceptions.py

```diff
@@ -0,0 +1,136 @@
+class Amazon_AdsError(Exception):
```

Comment:
Use CamelCase class names, remove underscore from all class names e.g. `Amazon_AdsError` to `AmazonAdsError`

```suggestion
class AmazonAdsError(Exception):
```

Reply:
Addressed review comment.

Reply:
There are some instances in ReadMe as well.

Reply:
Can you please elaborate this more.

Reply:
Sorry it's not related but still needs to be updated.

https://github.com/singer-io/tap-amazon-ads/blob/d19989d040f2727e3870c689ffa0bbda75852e27/README.md?plain=1#L291

Reply:
done